frontend/public/components/build-config.tsx

```diff
@@ -337,32 +386,29 @@ export const BuildConfigsList: React.FCC<BuildConfigsListProps> = (props) => {
     [builds, buildsLoaded, buildsLoadError],
   );
 
-  const buildResource = props.data
-    ? props.data.map((buildConfig) => {
+  const buildResource = data
+    ? data.map((buildConfig) => {
         buildConfig.latestBuild =
-          data.builds.latestByBuildName[
+          buildData.builds.latestByBuildName[
             `${buildConfig.metadata.name}-${buildConfig.metadata.namespace}`
           ];
         return buildConfig;
       })
     : [];
```

Comment:
```suggestion
const buildResource = data
    ? data.map((buildConfig) => ({
        ...buildConfig,
        latestBuild: buildData.builds.latestByBuildName[
          `${buildConfig.metadata.name}-${buildConfig.metadata.namespace}`
        ],
      }))
    : [];
```

Reply:
@sg00dwin, can you help me understand the need for this proposed change since it differs from the pre-existing code?